### DISCUSSION TITLE
某些书签（？）不显示，所以没法操作

### DISCUSSION THREAD

**Discussion by retsyo** [wmjordan/PDFPatcher#226] [问答] [answered]
# 情况描述
annas站点，很多“读秀”的图书，在 SumatraPDF 中打开的时候，可以看到在“书签”里面，有2项
```
annas_archive_metadata.json.zst
original_files.tar.zst
```
误点击的话，弹出“另存为”对话框

但是foxit、wps，打开，都说“没有书签”

# 需求解决方法
有啥办法，可以显示
```
annas_archive_metadata.json.zst
original_files.tar.zst
```
然后允许我删除？

可供参考的例子：
https://annas-archive.se/md5/790a71f5a5e6866b99bf9305d6058a8f

感谢

**Comment by wmjordan** [comment, marked as answer]
链接无法访问。这些可能是文档的附件。
请下载最新版，在清理选项中选中清除文档附件。

**Reply by retsyo** [reply]
“清除附件”解决了问题，谢谢
